paper-toggle-button.html

```diff
@@ -155,7 +155,7 @@
       }
     </style>
 
-    <div id="toggleContainer">
+    <div id="toggleContainer" class="toggle-container">
```

Comment:
I don't think this is used anywhere, other than for this previous styling, so I think you could remove the id altogether :)